compiler/main/driver.cpp

```diff
@@ -1367,6 +1368,38 @@ static void postVectorize() {
   }
 }
 
+//
+// TODO: Multi-Locale Interop prototype code!
```

Comment:
Nor this.  Please do a check for additional cases, I said there were more than just the ones I explicitly called out.

Reply:
This does not exist?

Reply:
My bad!

Reply:
These should all be addressed.